test/es-module/test-esm-loader-with-syntax-error.mjs

```diff
@@ -0,0 +1,22 @@
+import '../common/index.mjs';
+import { fileURL } from '../common/fixtures.mjs';
+import { match, ok } from 'assert';
+import { spawn } from 'child_process';
+import { execPath } from 'process';
+
+const child = spawn(execPath, [
+  '--experimental-loader',
+  fileURL('es-module-loaders', 'syntax-error.mjs').href,
+  fileURL('print-error-message.js').href,
```

Comment:
Same here, should be a path, can be omitted 
```suggestion
```

Reply:
It can be a path, sure. We can’t omit the entry, though, because part of the test is that the error message in `print-error-message.js` never gets printed. We’re testing that the error with the loader stops the startup process, rather than Node continuing on and evaluating the entry point. That’s what the `ok(!stderr.includes('Bad command or file name'));` is for.